Use Cases/SDPi/Temp from Repo/tf1-ch-c-use-cases.adoc

```diff
@@ -0,0 +1,88 @@
+= Appendix C – Device Point-of-care Interoperability (DPI) Use Cases
+
+
+NOTE:  This file integrates the use cases that are linked to the SDPi profiles to provide functional requirements, including any introductory sections for how use cases are managed in SDPi. +
+{empty} +
+Note that each use case is in its own file. +
+{empty} +
+Note that AsciiDoc Gherkin (plugin?) may be used ... eventually ... !
+
+[appendix#vol1_appendix_c_dpi_use_cases,sdpi_offset=C]
+== Appendix C - Device Point-of-care Interoperability (DPI) Use Cases
+
+This Appendix describes a number of use cases that explore the various SDPi functions.
```

Comment:
This line is duplicated in the line after.

Reply:
Not sure of the specific edit, but did remove the "Appendix C -" from both this == and the file initial = line.
QUESTION:  Does this appendix file need both the = and the == lines?  They get duplicated - at different levels - in the generated file.